src/IRremoteESP8266.h

```diff
@@ -882,6 +882,13 @@
 #define SEND_HAIER_AC160    _IR_ENABLE_DEFAULT_
 #endif  // SEND_HAIER_AC160
 
+#ifndef DECODE_TOTO
+#define DECODE_TOTO  _IR_ENABLE_DEFAULT_
```

Comment:
Short protocol name, should we keep indentation of _IR.... ?

Reply:
Oops. Yes we should. Will fix.

Reply:
Done